unepwcmc/bip
# Ticket 4: Join BIP component

## [Ticket 4: Join BIP component](https://unep-wcmc.codebasehq.com/projects/bip-website/tickets/4)

### Task

Add the Join Bip component to the homepage as shown in the designs.

![image](https://user-images.githubusercontent.com/56026099/203037215-b549c4bf-51de-4e36-8bab-32a64c202ddf.png)

### Testing

1. `git checkout feat/4-join-bip-component`. `git pull`
2. Spin up the project as usual
3. Visually inspect the component on the homepage. Are there any obvious changes which should be made?
4. Test the link in the component. It should redirect to `about/join-the-bip`.
5. Check the code changes — is there a clearer, simpler, more conventional or more re-usable way of making those changes?

Don't forget to update the ticket after the code review :)
